course6/toyapp_ariel/displaylib/primitive.cc

```diff
@@ -0,0 +1,6 @@
+#include "primitive.h"
```

Comment:
This file is not compiled, not added to CMake.
If it were, it wouldn't compile, as the method it defines is not declared anywhere.

Reply:
Removed